pkg/reconciler/dependencybuild/dependencybuild_test.go

```diff
@@ -99,7 +104,7 @@ func TestStateNew(t *testing.T) {
 			Name:      "test",
 		}, &db))
 		g.Expect(db.Status.CurrentBuildRecipe).ShouldNot(BeNil())
-		g.Expect(db.Status.CurrentBuildRecipe.Image).Should(Equal("quay.io/redhat-appstudio/hacbs-jdk11-builder:latest"))
+		g.Expect(db.Status.CurrentBuildRecipe.Image).Should(HavePrefix("quay.io/redhat-appstudio/hacbs-jdk"))
```

Comment:
this is a tweak I need to make to deal with the fact that the build recipe ordering is variable

Reply:
This is a bit of a regression, previously the ordering was defined by the 'builder-image.names' config map key. We probably need to add some kind of priority concept here.